CODEOWNERS

```diff
@@ -1,5 +1,23 @@
-# Code in the `rasa.shared` package is potentially re-used by downstream dependencies
-# such as Rasa X. Hence, changes within this package require double checking.
-/rasa/shared/ @RasaHQ/enable-squad
 /docs/docs/prototype-an-assistant.mdx @ricwo @alwx
```

Comment:
`/docs/docs/prototype-an-assistant.mdx @alwx`

Should we remove Rick as he left?

Reply:
maybe I can tag the squad instead of individuals to avoid this situation when they leave?